DevVehicle35-Motors/App/BusInteraction.cs

```diff
@@ -0,0 +1,59 @@
+﻿using DevVehicle35_Motors.Models;
+
+namespace DevVehicle35_Motors.App
+{
+    internal class BusInteraction
+    {
+        internal static void BuildBus()
+        {
+            Bus busan = new ();
```

Comment:
variable busan should be self-descriptive.

Reply:
Thank you for the recomendation Alejandro! I'm going to solve it.